components/uniset/src/builder.rs

```diff
@@ -378,6 +378,14 @@ impl UnicodeSetBuilder {
     pub fn complement_set(&mut self, set: &UnicodeSet) {
         self.complement_list(set.as_inversion_list());
     }
+
+    /// Returns whether the build is empty.
+    ///
+    /// Note: UnicodeSetBuilder panics when trying to build an empty builder.
```

Comment:
This sounds like a bug.  We should never panic if a problem is caused by user error.  Fix the bug rather than working around the bug.

Reply:
Done.